gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -36,7 +36,8 @@ def open(vm_disk)
     stat_script = <<-STAT_EOL
     (Get-Item "#{@virtual_disk}").length
 STAT_EOL
-    file_size, _stderr   = @parser.parse_single_powershell_value(@winrm.run_powershell_script(stat_script))
+    file_size, stderr   = @parser.parse_single_powershell_value(@winrm.run_powershell_script(stat_script))
+    raise "Unable to obtain virtual disk size for #{vm_disk}" if stderr =~ /At line:/
```

Comment:
I can certainly change that but for consistency's sake, there are a bunch of similar checks in the 3 modules in this directory which all use "=~"  - shouldn't they all do the same thing?